lb_content_resolver/unresolved_recording.py

```diff
@@ -122,3 +126,19 @@ def print_releases(self, releases):
             for rec in release["recordings"]:
                 print("   %-57s %d lookups" % (rec["recording_name"][:56], rec["lookup_count"]))
             print()
+
+    def cleanup(self):
+        """
+            Check the local collection and remove any recordings from unresolved recordings that have
+            been added to the DB.
+        """
+
+        query = f"""DELETE FROM unresolved_recording
```

Comment:
I am not sure how unresolved recording work but it seems weird to have a recording_mbid in column in it so probably double check this works as expected.

Reply:
Yes, that is correct. Since these recordings are not resolved, there are no entries in the Recording table for it, so we need to store the recording_mbid.